client-interfaces/src/main/java/ru/qa/api/client/enums/HttpMethod.java

```diff
@@ -4,4 +4,9 @@
  * Методы для работы с HTTP-протоколом
  */
 public enum HttpMethod implements Method{
+    GET;
+
+    public boolean isGet() {
+        return this.equals(GET);
```

Comment:
Это ж `enum`, тут можно и через `==`
```suggestion
        return this == GET;
```